upup/pkg/fi/nodeup/command.go

```diff
@@ -37,7 +38,7 @@ import (
 )
 
 // We should probably retry for a long time - there is not really any great fallback
-const MaxAttemptsWithNoProgress = 100
+const MaxTaskDuration = 365 * 24 * time.Hour
```

Comment:
Yeah, see the comment above it. In the case of `nodeup` there isn't much to do. I'm replacing a roughly 50m timeout with a more-or-less infinite timeout (there isn't a good constant for "infinite duration").